Changelog.md

```diff
@@ -12,8 +12,8 @@ Language Features:
 Compiler Features:
 * EVM: Support for the EVM Version "Cancun".
 * SMTChecker: Support `bytes.concat` except when string literals are passed as arguments.
-* TypeChecker: Comparison of internal function pointers now yields a warning, as it can produce unexpected results with the legacy pipeline enabled.
 * Standard JSON Interface: Add experimental support to import EVM assembly in the format used by ``--asm-json``.
+* TypeChecker: Comparison of internal function pointers now yields a warning, as it can produce unexpected results with the legacy pipeline enabled.
```

Comment:
Huh, what's wrong with the list indent? I just moved the entry down.

Reply:
```suggestion
 * TypeChecker: Comparison of internal function pointers now yields a warning, as it can produce unexpected results with the legacy pipeline enabled.
```

Reply:
It looks in-line with the rest of them though.

Reply:
It's differently indented. `*` vs ` *`. Look at the source, not the HTML preview.

Reply:
Ah OK, so it's for the whole Compiler Features section.